certbot/certbot/interfaces.py

```diff
@@ -368,26 +502,35 @@ def recovery_routine():  # type: ignore
 
         """
 
-    def config_test():  # type: ignore
+    @abstractmethod
+    def config_test(self) -> None:
         """Make sure the configuration is valid.
 
         :raises .MisconfigurationError: when the config is not in a usable state
 
         """
 
-    def restart():  # type: ignore
+    @abstractmethod
+    def restart(self) -> None:
         """Restart or refresh the server content.
 
         :raises .PluginError: when server cannot be restarted
 
         """
 
 
-class IDisplay(zope.interface.Interface):
+class IDisplay(zope.interface.Interface):  # pylint: disable=inherit-non-class
+    """Deprecated, use certbot.interfaces.Display as ABC instead."""
+
+
+@zope.interface.implementer(IDisplay)
+class Display(metaclass=ABCMeta):
```

Comment:
We still need the actual list of the methods to mock and freeze while creating the `FreezableMock` object in `_create_display_util_mock*`. What about using `certbot._internal.display.obj.FileDisplay` for that purpose instead of `certbot.interfaces.Display`?

Reply:
Ah I see. I was thinking that the object needed the properties of `certbot.display.util` but we're still mocking the `IDisplay` object and they differ slightly.

I think using `FileDisplay` is fine. Good idea.

Reply:
Ok. I did that, and also removed the decorator on the two internal display classes `FileDisplay` and `NonInteractiveDisplay`.